src/model.jl

```diff
@@ -92,28 +97,27 @@ $(TYPEDSIGNATURES)
 
 Returns a new `OptimalControlModel` instance, that is a model of an optimal control problem.
 
-The model is defined by the following arguments:
+The model is defined by the following optional keyword argument:
 
 - `time_dependence`: either `:autonomous` or `:nonautonomous`. Default is `:autonomous`.
-- `dimension_usage`: either `:scalar` or `:vectorial`. Default is `:scalar`.
 
 # Examples
 
 ```jldoctest
 julia> ocp = Model()
 julia> ocp = Model(time_dependence=:nonautonomous)
-julia> ocp = Model(dimension_usage=:vectorial)
-julia> ocp = Model(time_dependence=:nonautonomous, dimension_usage=:vectorial)
 ```
 
 !!! note
 
     - If the time dependence of the model is defined as nonautonomous, then, the dynamics function, the lagrange cost and the path constraints must be defined as functions of time and state, and possibly control. If the model is defined as autonomous, then, the dynamics function, the lagrange cost and the path constraints must be defined as functions of state, and possibly control.
```

Comment:
@ocots why "possibly control"? dynamics, lagrange objective and mixed constraints are always functions of the control